python-runnables/export-to-stages/runnable.py

```diff
@@ -0,0 +1,106 @@
+# This file is the actual code for the Python runnable export-to-stages
+import logging
+from dataiku import SQLExecutor2, Dataset, set_default_project_key
+from dataiku.runnables import Runnable
+
+
+class MyRunnable(Runnable):
```

Comment:
Do we have to call the call `MyRunnable` or could we use a more meaningful name?